swap/htlc.go

```diff
@@ -204,6 +221,50 @@ func swapHTLCScript(cltvExpiry int32, senderHtlcKey,
 	return builder.Script()
 }
 
+// swapHTLCScriptV2 returns the on-chain HTLC V2 witness script.
```

Comment:
maybe update the name and documentation of the V1 method also

Reply:
done